ui-schema.json

```diff
@@ -69,6 +78,9 @@
         "show": {
           "type": "string"
         },
+        "area": {
```

Comment:
Why have a separate `area` field. Area just maps to the key name in the layout matrix so can we just use the fields key from the schema?

Reply:
redacted, I now realize that field key is a json pattern match and area is okay for now

Reply:
It's basically there as an escape hatch. In case the inferred key name isn't a valid CSS grid-area name, or conflicts with another field with the same inferred key name. (I like the "where" key idea you mentioned in Slack... we'll keep it in mind for Formtron 2.0 since it would be a breaking change.)